packages/eslint-plugin/ROADMAP.md

```diff
@@ -96,7 +101,9 @@ It lists all TSLint rules along side rules from the ESLint ecosystem that are th
 | [`prefer-object-spread`]             | 🌟  | [`prefer-object-spread`][prefer-object-spread]                        |
 | [`radix`]                            | 🌟  | [`radix`][radix]                                                      |
 | [`restrict-plus-operands`]           | ✅  | [`@typescript-eslint/restrict-plus-operands`]                         |
+| [`static-this`]                      | 🛑  | N/A                                                                   |
 | [`strict-boolean-expressions`]       | ✅  | [`@typescript-eslint/strict-boolean-expressions`]                     |
+| [`strict-string-expressions`]        | 🛑  | N/A                                                                   |
```

Comment:
this is covered by two rules together:
`restrict-plus-operands` and `restrict-template-expressions`